src/Cake.Common.Tests/Unit/IO/FileAliasesTests.cs

```diff
@@ -200,6 +202,22 @@ public void Should_Copy_File()
                 fixture.TargetFiles[0].Received(1).Copy(
                     Arg.Is<FilePath>(p => p.FullPath == "/Working/target/file1.txt"), true);
             }
+
+            [Fact]
+            public void Should_Log_Verbose_Message_With_Correct_Target()
+            {
+                // Given
+                var fixture = new FileCopyFixture();
+
+                // When
+                FileAliases.CopyFile(fixture.Context, "./file1.txt", "./target/file1.txt");
```

Comment:
I agree, but the fixture isn't very straightforward in terms of changing that part.  I did, however, start from a failing test.
I can still try to get that to work if you like.